docs/source/usage/picmi/intro.rst

```diff
@@ -113,13 +113,31 @@ Parameters/Methods prefixed with ``picongpu_`` are PIConGPU-exclusive.
 
 - **Simulation**
 
-  - ``__init__(..., picongpu_template_dir)``:
+  additional constructor/configuration options:
+  - ``picongpu_template_dir``:
     Specify the template dir to use for code generation,
     please refer to :ref:`the documentation on the matter for details <picmi-custom-generation>`
-  - ``__init__(...,  picongpu_typical_ppc)`` typical ppc to be used for normalization in PIConGPU
+  - ``picongpu_typical_ppc``:
+    typical ppc to be used for normalization in PIConGPU, if not set explicitly, PIConGPU will use the median ppc of all defined species
+  - ``picongpu_moving_window_move_point``:
+    portion of the simulation window a light ray reaches from the time of the start of the simulation until the simulation window begins to move.
```

Comment:
This description reads somehow convoluted (although probably technically correct). Is there another piece of documentation about this feature in the pure PIConGPU docs that you could refer to? If not, we should create that.

Reply:
Not as far as I know in our out of code documentation, as far as I know the option is only documented in the ``--help`` option of picongpu and some example ``.cfg``'s.

I definitely agree that we should add documentation for this in a separate PR